docs/UserGuide.md

```diff
@@ -340,29 +430,43 @@ tags contain `Thursday` will be listed out.
   
 #### 3.5 Editing a person : `c-edit`
 
+Edits the corresponding employee's information in the Employee Directory.
 
-Edits the corresponding contact information in the contact list.
-
-Format: `c-edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMERGENCY_CONTACT] [a/ADDRESS] [t/TAG] ...`
+Format: `c-edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMERGENCY_CONTACT] [a/ADDRESS] [t/TAG]`
 
-* Edits the employee at the specified `INDEX`. The index refers to the index number shown in the displayed contact list. The index **must be a positive integer** 1, 2, 3, ...​
+* Edits the employee at the specified `INDEX`. The index refers to the index number shown in the displayed Employee Directory. The index must be a positive integer 1, 2, 3, ...
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the employee will be removed i.e adding of tags is not cumulative.
-* You can remove all the employees’s tags by typing `t/` without
-    specifying any tags after it.
+* When editing tags, the existing tags of the employee will be removed i.e. adding of tags is not cumulative.
+* You can remove all the employee's tags by typing `t/` without specifying any tags after it.
+
+Example: 
+
+Let's say you want to edit an employee whose index is 1 in Employee Directory. His new phone number is 91234567. His new emergency contact is 81232744. 
+You can follow these instructions:
 
-Examples:
-*  `c-edit 1 p/91234567 e/81234567` Edits the phone number and emergency contact of the 1st employee to be `91234567` and
- `81234567` respectively.
-*  `c-edit 2 n/Betsy Crower t/` Edits the name of the 2nd employee to be `Betsy Crower` and clears all existing tags.
 
+1. Type `c-edit 1 p/91234567 e/81232744` into the _Command Box_.
+1. Press "Enter" to execute.
 
-#### 3.6 Locating persons by keywords: `c-find`
 
-Finds all active(unarchived) contacts that contain the KEYWORD(s) in their names.
+Outcome:
+1. The _Result Display_ will show that you edit the employee.
+
+2. All active employees will be listed in the _Main View_.
+
+Before executing:
+![BeforeEditCommandScreenshot](images/BeforeClearCommand.png)
+
+After executing:
+![EditCommandScreenshot](images/EdirCommandResult.png)
```

Comment:
Add a blank line between executing and the image, so that they appear on different lines